src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -783,7 +783,7 @@ namespace cryptonote
     if(bvc.m_added_to_main_chain)
     {
       cryptonote_connection_context exclude_context = boost::value_initialized<cryptonote_connection_context>();
-      NOTIFY_NEW_BLOCK::request arg = AUTO_VAL_INIT(arg);
+      NOTIFY_NEW_FLUFFY_BLOCK::request arg = AUTO_VAL_INIT(arg);
```

Comment:
That's another big one! Thanks again for taking the time to go through everything. Much appreciated.